packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -300,14 +300,14 @@ class WebView extends StatefulWidget {
   /// By default `debuggingEnabled` is false.
   final bool debuggingEnabled;
 
-  /// The value used for the HTTP User-Agent: request header.
   /// A Boolean value indicating whether horizontal swipe gestures will trigger back-forward list navigations.
   ///
   /// This only works on iOS.
   ///
   /// By default `gestureNavigationEnabled` is false.
   final bool gestureNavigationEnabled;
 
+  /// The value used for the HTTP User-Agent: request header.
```

Comment:
I don't know wether we need to open another PR to merge this, I think it's trivial. If you think we need, I can revert this. :)

Reply:
I'm fine with it =)